src/file_format.rst

```diff
@@ -0,0 +1,189 @@
+
+In an empty directory, open a text file and copy the following text inside:
+
+.. code-block:: LAMMPS
+
+   lattice sc 1.
+   region box block 0 5 0 5 0 5
+   create_box 1 box
+   create_atoms 1 box
+
+   mass 1 1
+
+   write_data data.lmp
+   write_dump all atom dump.lammpstrj
+
+Let's save the file with the name `in.lmp`. Now in the command-line you can
+run the following command to execute LAMMPS with this input:
+
+.. code-block:: bash
+
+   lmp -i in.lmp
+
+You should see a bunch of lines appear in your screen. The first one should start
+with `LAMMPS` followed by a parentheses with the specification of the version
+of the code you are using. The last line should read something like
+`Total wall time: 00:00:00`. If you've never executed LAMMPS before,
+congratulation! This is maybe your first successful (very simple) simulation!
+
+You will also notice that two files appeared in the directory: `data.lmp` and
+`dump.lammpstrj`. Let's start by opening the first one.
+
+Data file format
+****************
+
+The first file of the two is usually referred to as a `data file`. Its format
+is rather strict and you can see that there are general information at the
+top of the file and several sections starting with a capitalized word, a blank
+line and some numbers. Let's slowly go through all of this.
+
+The first part of the file is called `the header`. The first line of the file
+is always a comment that LAMMPS does not read but which can contain
+information on the way the file was generated. Here we see the LAMMPS version
+and some more information like timestep and units. The following lines contain
+the number of `atoms` (125), the number of `atom types` (1) and three lines
+containing `xlo xhi` like indications. This header is simple, but generally,
+headers can contain much more information. The first two lines are explicit,
+you define a system with 125 atoms all of which have the same characteristics.
+The last three lines define the volume in which these atoms are contained. It
+is a box with edge coordinates starting at 0 and ending at 5 in every direction
+(x, y and z).
+
+From here starts the body of the file. The order of the sections is not important
+but all of them must come in the format:
+
+.. code-block:: LAMMPS
+
+   Section name # Capitalized, correctly spelled
+   # blank line
+   Section input # Number of line and format depend on the section.
+
+The first section you should see is the `Masses` section. In LAMMPS, masses are
+assigned to atom types so you only have one line here. All types have their own
+masses but several types can have the same mass. Types are LAMMPS way to refer
+to properties of particles that are *the same* for all particles of *the same
+type*. They are also used to determine how particles interact with one another.
+Types are not always binded to chemical species and you will see in further
+tutorials how it can be convenient to define different types for similar atoms.
```

Comment:
```suggestion
usually assigned by atom types. Since you have only one atom type
in this simulation, this `Masses` section has only one line.
```

"usually" added because LAMMPS can have per-atom `rmass` (unlike GROMACS for example).